src/TourPortal.js

```diff
@@ -97,6 +99,9 @@ class TourPortal extends Component {
       inDOM: false,
       observer: null,
     }
+
+    const { skipNotFound, steps } = this.props
+    this.steps = skipNotFound ? this.filterSteps(steps) : steps
```

Comment:
I'm not sure if is the right place to check/filter if DOM nodes are present.

Reply:
I am sorry but I do not understand well either.